docs/monitoring-usage.rst

```diff
@@ -278,3 +278,86 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Metrics
+---------------------------
+
+The Stackdriver Monitoring API can be used to write data points to custom metrics. Please refer to
+the documentation for the `Creating Custom Metrics`_ for more information.
+
+To create a custom :class:`~gcloud.monitoring.timeseries.TimeSeries` value, you must first create a
+custom :class:`~gcloud.monitoring.metric.MetricDescriptor` object, and then use its
+type to create a fully parameterized :class:`~gcloud.monitoring.metric.Metric` by providing
+specific values for the available labels. For more information on creating metric descriptors,
+see the `Metric Descriptors`_ section.
```

Comment:
I attempt to address it, so if you have any other improvements in mind will need you to be more specific. I will note that this comment is on the outdated commit.